compiler-cli/src/main.rs

```diff
@@ -396,11 +395,11 @@ fn command_build(
         .map(|_| ());
     }
 
-    diagnostic::write_title(
-        &mut buffer,
-        "Deprecated rebar3 build command",
-        Severity::Warning,
-    );
+    // diagnostic::write_title(
+    //     &mut buffer,
+    //     "Deprecated rebar3 build command",
+    //     Severity::Warning,
+    // );
```

Comment:
We want to keep this

Reply:
It seem the function has been removed, so I commented that part

Reply:
Yes that's intentional. This isn't finished yet so it should not compile. The errors are there to guide development, to ensure nothing is missed.